pkg/operation/botanist/component/resourcemanager/resource_manager_test.go

```diff
@@ -2160,8 +2196,6 @@ subjects:
 	Describe("#Destroy", func() {
 		Context("target differs from source cluster", func() {
 			JustBeforeEach(func() {
-				configMap = configMapFor(&watchedNamespace, pointer.String(gardenerutils.PathGenericKubeconfig))
-				deployment = deploymentFor(configMap.Name, cfg.KubernetesVersion, &watchedNamespace, pointer.String(gardenerutils.PathGenericKubeconfig), true, nil)
```

Comment:
Why was this removed?

Reply:
The vars are not used for the tests that test the `#Destroy` func. The expectations in the tests below are in the form of 
```go
						c.EXPECT().Delete(ctx, &appsv1.Deployment{ObjectMeta: metav1.ObjectMeta{Namespace: deployNamespace, Name: "gardener-resource-manager"}}),
```